component/block_cache/block_cache.go

```diff
@@ -261,14 +264,21 @@ func (bc *BlockCache) Configure(_ bool) error {
 				return fmt.Errorf("config error in %s [%s]", bc.Name(), err.Error())
 			}
 		}
+		var stat syscall.Statfs_t
+		_ = syscall.Statfs(bc.tmpPath, &stat)
```

Comment:
Same as previous comment. Log the error here and assign diskSize with some default value.